tests/lib/processor.js

```diff
@@ -199,7 +199,7 @@ describe("processor", () => {
 
             assert.strictEqual(blocks.length, 1);
             assert.strictEqual(blocks[0].filename, "0.js");
-            assert.strictEqual(blocks[0].text, "\n\n \n  \n");
+            assert.strictEqual(blocks[0].text, "\n\n\n \n  \n");
```

Comment:
Aha! This reminded me of both #77 and #78. After some digging, it turns out [`remark-parse@7.0.0`](https://github.com/remarkjs/remark/releases/tag/remark-parse%407.0.0) includes https://github.com/remarkjs/remark/pull/423, which fixes both of those issues. I opened #175 with a few additional tests specifically demonstrating those fixes.